extension/chrome/elements/pgp_block_modules/pgp-block-render-module.ts

```diff
@@ -328,6 +328,9 @@ export class PgpBlockViewRenderModule {
           image.addEventListener('load', () => this.resizePgpBlockFrame());
         }
       } catch (e) {
+        // When CORS error, set error message to `Remote server blocked image loading`
+        const errorMessage = e.message?.includes('Failed to fetch') ? 'Remote server blocked image loading' : 'Failed to load image';
```

Comment:
Firefox currently shows `Failed to load image` for CORS error, as it has different `e.message` - `NetworkError when attempting to fetch resource`

Reply:
Ah, it seems `Failed to fetch` and `NetworkError when attempting to fetch resource` errors are shown not only for CORS errors, but for other cases too, like bad internet connection or some network error.

Then let's show `Failed to load image` for all errors

Reply:
Sounds good